easybuild/easyconfigs/a/annovar/annovar-2019Oct24-GCCcore-8.2.0-Perl-5.28.1.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Tarball'
+
+name = 'annovar'
+# Unconventional version is taken from the upstream site
+# https://doc-openbio.readthedocs.io/projects/annovar/en/latest/user-guide/download/
+version = '2019Oct24'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://annovar.openbioinformatics.org/en/latest/'
+description = """ ANNOVAR is an efficient software tool to utilize update-to-date information 
+ to functionally annotate genetic variants detected from diverse genomes (including human 
+ genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others)."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+# Registration required for download
+# http://download.openbioinformatics.org/annovar_download_form.php
+# rename after download to %(name)s-%(version)s.tar.gz
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['4061af3f2cc8f3d4d8187986c3d468a4b6ae145eec1858e85c7cd1059debb8ed']
+
+dependencies = [('Perl', '5.28.1')]
+
+modextrapaths = {
+    'PATH': '',
+}
+
+sanity_check_paths = {
+    'files': ["annotate_variation.pl", "retrieve_seq_from_fasta.pl"],
```

Comment:
Can we include a simple sanity check command for at least one of these scripts?
Also, do they use the right `perl` command (`/usr/bin/env perl`)?

Reply:
I could add something like 
```
annotate_variation.pl --help 2>&1 | grep Usage
```
as a sanity check for all scripts. Would this be OK?

Reply:
Forgot to mention that all scripts use `/usr/bin/env perl` .

Reply:
@seb45tian ping?
the sanity check command you suggest is ok (although grepping a line a bit further down would be slightly better).

Reply:
All done. Thanks for the reminder.